### PR TITLE
qgroundcontrol: Add version 4.0.6

### PR DESCRIPTION
**Pull request by TheZoker**
Adds [QGroundControl ](http://qgroundcontrol.com/) version 4.0.6

### CODE REVIEW

**Comment by TheZoker** [comment]
@r15ch13 Can you help me here please? Thanks!

File: bucket/qgroundcontrol.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    "shortcuts": [
+        [
+            "qgroundcontrol.exe",
+            "QGroundControl"
+        ]
+    ],
+    "checkver": "github",
+    "autoupdate": {
+        "url": "https://github.com/mavlink/qgroundcontrol/releases/download/$version/QGroundControl-installer.exe"
+    }
```
Comment (Ash258) [inline bucket/qgroundcontrol.json:18]:
```suggestion
    "version": "4.0.6",
    "description": "Intuitive and Powerful Ground Control Station for the MAVLink protocol",
    "homepage": "http://qgroundcontrol.com",
    "license": "Apache-2.0|GPL-3.0-only",
    "suggest": {
        "vcredist": "extras/vcredist2019"
    },
    "architecture": {
        "64bit": {
            "url": "https://github.com/mavlink/qgroundcontrol/releases/download/v4.0.6/QGroundControl-installer.exe#/dl.7z",
            "hash": "76dd7acaf6c5d6d3de255d88fda0197a29aab00ab1f62d90d085e81d4fe8daf3"
        }
    },
    "pre_install": "Remove-Item \"$dir\\`$*\", \"$dir\\vc_red*\", \"$dir\\*uninsta*exe\" -Recurse",
    "bin": "QGroundControl.exe",
    "shortcuts": [
        [
            "QGroundControl.exe",
            "QGroundControl"
        ]
    ],
    "checkver": {
        "github": "https://github.com/mavlink/qgroundcontrol"
    },
    "autoupdate": {
        "architecture": {
            "64bit": {
                "url": "https://github.com/mavlink/qgroundcontrol/releases/download/v$version/QGroundControl-installer.exe#/dl.7z"
            }
        }
    }
```

Comment (TheZoker) [reply bucket/qgroundcontrol.json:18]: That's what's on the website:
Screenshot_2020 05 07_00h41m28s_001_

Source: http://qgroundcontrol.com/

**Comment by Ash258** [comment]
> Where did you get information about license?

**Comment by TheZoker** [comment]
@Ash258 I answered that in the resolved conversation:
https://github.com/lukesampson/scoop-extras/pull/3985#discussion_r421134179

Quote:

> That's what's on the website:
> Screenshot_2020 05 07_00h41m28s_001_
> 
> Source: http://qgroundcontrol.com/

**Comment by Ash258** [comment]
Then stop marking conversations as resolved, when there is still discussion going....

https://github.com/lukesampson/scoop-extras/pull/3056#discussion_r338548603

**Comment by TheZoker** [comment]
Sorry, I didn't click on the "Resolve" button on purpose 😅 

Is there anything that's needs to be done, for this PR to get merged?

**Pull request merged**